### PR TITLE
added like count to tweets..refreshed colors

### LINKED ISSUE #97: Update tweets so it shows the number of likes
The tweet should show the number of likes next to the like button

### PR DESCRIPTION
**Pull request by lmurdock12**
Closes #97

### CODE REVIEW

File: MainApp/models.py [hunk 1]
```diff
@@ -25,7 +25,11 @@ def visible_info(self):
         if not self.tweet_text:
             return
         return self.tweet_text
-    
+
+    def countLikes(self):
+        return len(Like.objects.filter(tweet=self))
+
+    likes = property(countLikes)
```
Comment (lmurdock12) [inline MainApp/models.py:32]: This added a property to get the number of likes to the current tweet.
Whenever we are referencing a tweet object in a django template we can just use tweet.likes to get the number of likes

File: MainApp/templates/MainApp/base_homepage.html [hunk 2]
```diff
@@ -252,17 +252,40 @@
...
+  border-style: solid;
+  border-radius: 32px;
+  background-color: rgba(0,0,0,0);
+  color: rgba(138, 5, 5, 0.747);
+  outline: none
+}
+
+.unlike:hover {
+  background-color: rgba(179, 10, 80, 0.15);
+  color: rgba(138, 5, 5, 0.747);
```
Comment (lmurdock12) [inline MainApp/templates/MainApp/base_homepage.html:288]: Refreshed the colors of the like/unlike buttons...it does not look the greatest right now but its a start. We can make it look better towards the end

File: MainApp/templates/MainApp/profile.html [hunk 3]
```diff
@@ -75,21 +75,23 @@ <h3>{{profile_user.bio}}</h3>
...
-       <form name="like_tweet" method="POST" action="{% url 'MainApp:homepage' %}">
-         {% csrf_token %}
-         <!--Show either like or unlike depending on whether or not user has liked the tweet-->
-         {% if not isLiked %}
-           <button name="like_button" value="{{user.username}},{{post.pk}}" class="like">Like</button>
-          {% else %}
-            <button name="unlike_button" value="{{user.username}},{{post.pk}}" class="like">Unlike</button>
-          {% endif %}
-        </form>
-        {% endifnotequal %}
```
Comment (lmurdock12) [inline MainApp/templates/MainApp/profile.html:90]: Remove the check to prevent the current authenticated user from liking their own tweet.

**Comment by lmurdock12** [comment]
Let me know if it looks good for yall.

**Review by natalie2by4 — APPROVED** [review]
Looks good

**Review by bhball22 — APPROVED** [review]
Looks great, if we have more time and want to change something this would be an easy reformatting design to do.

**Pull request merged**